core/src/apps/common/request_passphrase.py

```diff
@@ -53,7 +53,7 @@
 async def request_passphrase_ack(ctx: wire.Context, on_device: bool) -> str:
     if not on_device:
         text = Text("Passphrase entry", ui.ICON_CONFIG)
-        text.normal("Please, type passphrase", "on connected host.")
+        text.normal("Please type your", "passphrase on the", "connected host.")
         await Popup(text)
```

Comment:
The comma after "Please" emphasizes the word too much, like "Prosím vás! Zadejte ...":
<img src="https://user-images.githubusercontent.com/42678794/65315609-b1711180-db98-11e9-8e2d-2429430992fa.png" width=300><img src="https://user-images.githubusercontent.com/42678794/65315617-b5049880-db98-11e9-8ed5-5fb1da313fcd.png" width=300>